public/translation_fr.json

```diff
@@ -26,7 +26,9 @@
         "import-note": "Note: Les personnages importés auront des champs vide pour les objets et sessions, merci de les saisir manuellement.",
         "enter-details": "Ajouter des détails",
         "privacy-details": "Confidentialité",
-        "import-help-text-3": "Entrer votre lien de la fiche sur D&D Beyond."
+        "import-help-text-3": "Entrer votre lien de la fiche sur D&D Beyond.",
+        "create-success-message": "Personnage crée",
+        "edit-success-message": "Personnage mise à jour"
```

Comment:
I feel like this should say `Personnage mis à jour` instead.

Reply:
done